sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/AnalyticsTransmissionTarget.java

```diff
@@ -192,12 +236,27 @@ String getTransmissionTargetToken() {
     }
 
     /**
-     * Getter for property configurator to override Common Schema Part A properties.
-     *
-     * @return  the Property Configurator
+     * Init channel listener to add tickets to logs.
      */
-    public PropertyConfigurator getPropertyConfigurator() {
-        return mPropertyConfigurator;
+    static Channel.Listener getChannelListener() {
+        return new AbstractChannelListener() {
+
+            @Override
+            public void onPreparingLog(@NonNull Log log, @NonNull String groupName) {
+                addTicketToLog(log);
+            }
+        };
+    }
+
+    /**
+     * Add ticket to common schema logs.
```

Comment:
What is a ticket again, exactly?  Is "ticket" universal terminology that is used for other authentication frameworks?  If not, it's worth clarifying in comments.

Reply:
This is the right word in One Collector world.